packages/desktop-client/src/components/reports/reports/Spending.tsx

```diff
@@ -139,48 +139,115 @@ export function Spending() {
       <View
         style={{
           flexDirection: 'row',
+          alignItems: 'center',
+          padding: 20,
+          paddingBottom: conditions.length > 0 ? 0 : 10,
           flexShrink: 0,
         }}
       >
         <View
           style={{
-            padding: 20,
-            flexShrink: 0,
+            alignItems: 'center',
+            flexDirection: 'row',
+            marginRight: 5,
           }}
         >
-          {conditions && (
-            <View style={{ flexDirection: 'row' }}>
-              <FilterButton
-                onApply={onApplyFilter}
-                compact={false}
-                hover={false}
-                exclude={['date']}
-              />
-              <Tooltip
-                placement="bottom start"
-                content={<Text>Save compare and filter options</Text>}
-                style={{
-                  ...styles.tooltip,
-                  lineHeight: 1.5,
-                  padding: '6px 10px',
-                  marginLeft: 10,
-                }}
-              >
-                <Button
-                  type="primary"
-                  style={{
-                    marginLeft: 10,
-                  }}
-                  onClick={saveFilter}
-                  disabled={filterSaved ? true : false}
-                >
-                  {filterSaved ? 'Saved' : 'Save'}
-                </Button>
-              </Tooltip>
-              <View style={{ flex: 1 }} />
-            </View>
+          <strong
+            style={{
+              paddingRight: 5,
+            }}
+          >
+            Compare
+          </strong>
+          <Select
+            value={compare}
+            buttonStyle={{
+              fontWeight: 'bold',
+            }}
+            onChange={e => {
+              setCompare(e);
+              if (mode === 'lastMonth') setMode('twoMonthsPrevious');
+              if (mode === 'twoMonthsPrevious') setMode('lastMonth');
+            }}
+            options={[
+              ['thisMonth', 'this month'],
+              ['lastMonth', 'last month'],
+            ]}
+          />
+          <strong
+            style={{
+              paddingRight: 10,
+              paddingLeft: 5,
+            }}
+          >
+            to the:
+          </strong>
+          <ModeButton
+            color="inherit"
+            selected={['lastMonth', 'twoMonthsPrevious'].includes(mode)}
+            onSelect={() =>
+              setMode(
+                compare === 'thisMonth' ? 'lastMonth' : 'twoMonthsPrevious',
+              )
+            }
+          >
+            Month previous
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
            Previous month
```